config.js

```diff
@@ -4,6 +4,7 @@ require('dotenv').config();
 
 module.exports = {
   isProduction: process.env.NODE_ENV === 'production',
+  bocoupTeamId: 'T025GMFDP',
```

Comment:
I dunno. Not sure what we'd need it for, other than specific one-offs like this.